plugins/render/api_key.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+type Config struct {
+	Version           int
+	SSHPreserveHosts  bool `yaml:"sshPreserveHosts"`
```

Comment:
Similar to this field - is this required in order to successfully authenticate against the Render service?

Reply:
updated